Subject: remove the `config/includes.chroot/home/@USERNAME@` directory
Signed-off-by: Mattéo Rossillol‑‑Laruelle <beatussum@protonmail.com>
Description:
Remove the generic home directory before creating the live image.

--- a/auto/config
+++ b/auto/config
@@ -107,6 +107,7 @@ rm -f config/archives/kali-proposed-updates.list.*
 # Copy over all files from official kali configuration
 cp -rT kali-config/common config
 [ ! -d kali-config/variant-$variant ] || cp -rTL kali-config/variant-$variant config
+test "${variant}" = "bsum" && rm -r config/includes.chroot/home/@USERNAME@
 
 if [ -n "$enable_pu" ]; then
 	mkdir -p config/archives
